docs/guides/resources/user-account-resources.primary.in.rst

```diff
@@ -0,0 +1,168 @@
+It is a requirement defined by the `Science Requirements Document <https://docushare.lsst.org/docushare/dsweb/Get/LPM-17>`_ that the "Data Management System will also provide at least 10% of its total capability for user-dedicated processing and user-dedicated storage" (Section 3.5).
+
+This page quantifies the individual and shared resources that every account at the US Data Access Center (US DAC) has access to, by default, via the Rubin Science Platform at |rsp-url|.
+
+.. important::
+   The US DAC, the Rubin Science Platform, and this documentation are currently under construction.
+   The estimates quoted below include current, planned, and to-be-determined (TBD) values, all of which are subject to change as hardware, software, and user habits evolve.
+
+**Page last updated:** March 17 2025
+
+
+Computational processing
+========================
+
+Computational resources are available via the Notebook Aspect (JupyterLab).
+
+Notebook server options:
+
+- Small (1.0 CPU, 4 GB RAM)
+- Medium (2.0 CPU, 8 GB RAM)
```

Comment:
Sorry I keep neglecting to bring this up, but I am severely tempted to remove medium as an option. It seems almost nobody is using it and it risks confusing users who try to do things that run out of memory. Discuss this on slack?

(Also I might slightly tweak those values to better utilize those nodes but no worries, we can change the numbers later)